pkg/invariantlibrary/allowedbackenddisruption/matches_test.go

```diff
@@ -274,16 +275,20 @@ func TestDisruptionDataFileParsing(t *testing.T) {
 			foundMetalOVN = true
 		}
 	}
-
+	currentRelease = currentReleaseFromMap(releasesInQueryResults)
 	assert.Greater(t, dataOver100Runs, 5,
 		"expected at least 5 entries in query_results.json to have over 100 runs")
 	assert.True(t, foundAWSOVN, "no aws ovn job data in query_results.json")
 	assert.True(t, foundGCPOVN, "no gcp ovn job data in query_results.json")
 	assert.True(t, foundAzureOVN, "no azure ovn job data in query_results.json")
 	assert.True(t, foundMetalOVN, "no metal ovn job data in query_results.json")
-	assert.Equal(t, 1, len(releasesInQueryResults),
-		"expected only one Release in query_results.json")
-
+	if currentRelease <= "4.14" {
```

Comment:
Have already discussed with @deepsm007 to change these to use integer comparison logic and not strings.  Can evaluate using [getMinor](https://github.com/openshift/origin/blob/a4bbe975a9361503a929761b969bf3504225d6ae/pkg/invariantlibrary/historicaldata/next_best_guess.go#L123https://github.com/openshift/origin/blob/a4bbe975a9361503a929761b969bf3504225d6ae/pkg/invariantlibrary/historicaldata/next_best_guess.go#L123) and [getMajor](https://github.com/openshift/origin/blob/a4bbe975a9361503a929761b969bf3504225d6ae/pkg/invariantlibrary/historicaldata/next_best_guess.go#L115C9-L115C9) from next_best_guess.go